build/package/Installer.DEB.targets

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Target Name="SetupDebProps"
-          DependsOnTargets="Init">
+      DependsOnTargets="Init">
```

Comment:
And ideally if https://github.com/dotnet/codeformatter can work with our repo, this kind of conversion help improve the formatting config file instead of a one time fix. And I am working on it slowly